File: Ass1/evenodd.cs
using System;
class evenodd
{
    static void Main(string[] args)
    {
        int a;
        Console.WriteLine("Please input an integer: ");
        a = int.Parse(Console.ReadLine());
        Console.WriteLine("{0} is an {1} number.", a, a % 2 == 0 ? "even" : "odd");
    }
}

File: Ass1/evenodd.java
import java.util.Scanner;
public class evenodd{
    public static void main(String[] args){
        int a;
        Scanner scan = new Scanner(System.in);
        System.out.println("Please input an integer: ");
        a = scan.nextInt();
        scan.close();
        System.out.printf("%d is an %s number.\n", a, a % 2 == 0 ? "even" : "odd");
    }
}